FirebaseAppDistribution/Sources/FIRFADApiService.h

```diff
@@ -89,6 +89,6 @@ typedef NS_ENUM(NSUInteger, FIRFADApiError) {
   // Failure to parse Api response
   FIRApiErrorParseFailure = 7,
 
-} NS_SWIFT_NAME(AppDistributionApiError);
```

Comment:
Is this no longer needed now that we are implementing the service in Swift?

Reply:
I implemented AppDistributionApiError in Swift and so removed the duplicate name.

I'm working on a solution where I stop using the swift implementation in the tests, and instead create a duplicate set of tests identical to this one. That way this can be reverted.